docs/TuxAnimations.md

```diff
@@ -0,0 +1,208 @@
+# Tux/Animations
+
+##Premise
+>Tux Animations is a library of default animation components and            easings. Additionally Tux Animations includes the makeAnimation function     which can be used to create custom animation components.
+
+
+##Implementation
+####1) Requiring Animations Components
+The Tux Animations interface including all animation defaults can be exposed via:
+
+    var Animations = require('tux/Animations');
+
+#####Require only what you need
+Tux is an extremely modular-conscious framework and as such it is possible to require only the minimal number of files needed in order to save space in your app. For Example, if you need access to the default Fly and FadeUp components in your implementation, you need only to expose those specific animations via:
+
+    var Fly = require('tux/Animations/Fly');
+    var FadeUp = require('tux/Animations/Fade/Up');
+
+
+####2) Rendering Animation Components
+The only other step to implementing a Tux animation is to wrap the element you want animated with the desired animation component within the render method of your <code>React.createClass</code> function. All nested elements will then inherit the properties of the parent animation component.
+
+    render: function() {
+        return (
+            <Fly>
+                <h1>Hello World</h1>
+            </Fly>
+        );
+    }
+![Animation 1](http://cdn.makeagif.com/media/1-12-2015/WjxMrn.gif)
+#####Default Animations
+In this implementation, the Fly component will inherit it's default CSS transitions because no addtional properties are defined. Each animation component has its own default CSS transitions which you can view via their own documentation.
+
+####3) Customize Default Animation Properties
+Tux also allows easy customization of default animation components via their properties. Each animation component has 4 properties which are customizable.
+
+1. Duration
+2. Delay
+3. Easing
+4. Custom
+
+#####Duration Property
+The Duration property as the name would imply, determines the length of the animation sequence. Duration accepts a string with any standard CSS suffix or number input, but **beware** that numbers <code> will resolve in milliseconds:</code>
+
+    <Fly duration="2s">
+        <h1>Hello World</h1>
+    </Fly>  //or
+    <Fly duration={2000}>
+        <h1>Hello World</h1>
+    </Fly>
+
+#####Delay Property
+The Delay property sets a delay on the animation before it activates. The delay property is similar to the duration property in that it accepts either a string or number input.
```

Comment:
Add the following: `The following example will add a 1 second delay to our animation`.